myapp/app/views/tasks/_form.html.erb

```diff
@@ -8,7 +8,7 @@
   <%= f.input :name, input_html: { autofocus: true } %>
   <%= f.input :description, input_html: { autofocus: true } %>
   <%= f.input :priority, collection: Task.priority_types.keys.zip(Task.priority_types.values), selected: f.object.priority || 1, input_html: { autofocus: true } %>
-  <%= f.input :expired_date, input_html: { autofocus: true } %>
+  <%= f.input :expired_date, include_blank: true, input_html: { autofocus: true } %>
```

Comment:
Maybe don't need 'autofocus: true'.
need 'input :name' only.
 
https://so-prog-engineer.hatenablog.com/entry/2023/03/18/214517

Reply:
Thanks for the review!
I deleted unnecessary autofocus in 
Please kindly check it~ https://github.com/Fablic/training/pull/1566/commits/e6314e1048405c152f8f98e422cc78b461484702